packages/hostScriptsServices/hostScripts/recreate_dappnode.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Host script service executed with a timer to ensure dappmanager container is running, otherwise recreate dappnode to latest version
+# The edge case where the dappmanager container does not exist is not covered
```

Comment:
We could easily handle it, right? We could do `docker compose up` of the Dappmanager in DNCORE directory

Reply:
Yeah, will add it